src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -367,14 +367,18 @@ export default function DownloadDialog({
   }
 
   const readyToInstall =
-    installPath &&
-    gameInstallInfo?.manifest?.download_size &&
-    !gettingInstallInfo
+    gameInstallInfo?.game['name'] === 'web' ||
```

Comment:
Could we simplify this, perhaps splitting it into two or three variables?
Maybe double-check if one of these checks is used in other places and make them more readable.

Reply:
updated now :)